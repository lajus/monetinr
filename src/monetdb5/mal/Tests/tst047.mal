#testing copying and counting reversals
#to test reversal handling

b:= bat.new(:oid,:int);
bat.append(b,1);
bat.append(b,4);
bat.append(b,9);
io.print(b);
br:= bat.reverse(b);
io.print(br);

l0:= algebra.copy(b);
io.print(l0);
