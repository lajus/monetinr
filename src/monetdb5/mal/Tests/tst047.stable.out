stdout of test 'tst047` in directory 'monetdb5/mal` itself:


# 07:58:46 >  
# 07:58:46 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst047.mal </dev/null
# 07:58:46 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
##testing copying and counting reversals
##to test reversal handling
#
#b:=new(:int,:int);
#insert(b,1,1);
#insert(b,2,4);
#insert(b,3,9);
#print(b);
#br:= reverse(b);
#print(br);
#
#l0:= copy(b);
#print(l0);
#l1:= copy(br);
#print(l1);
#
function user.main():void;
#testing copying and counting reversals 
#to test reversal handling 
    b := bat.new(:oid,:int);
    bat.append(b,1);
    bat.append(b,4);
    bat.append(b,9);
    io.print(b);
    br := bat.reverse(b);
    io.print(br);
    l0 := algebra.copy(b);
    io.print(l0);
end main;
#-----------------#
# h	tmp_4	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  4	  ]
[ 2@0,	  9	  ]
#-----------------#
# t	tmp_4	  # name
# int	int	  # type
#-----------------#
[ 1,	  0@0	  ]
[ 4,	  1@0	  ]
[ 9,	  2@0	  ]
#-----------------#
# h	tmp_5	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  4	  ]
[ 2@0,	  9	  ]

# 07:58:46 >  
# 07:58:46 >  Done.
# 07:58:46 >  

